src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString
```

Comment:
The function starts by splitting the `sourceString` by semicolons, which is correct. However, ensure that the input string is not empty or null to avoid potential errors.